src/utils/validation.ts

```diff
@@ -76,3 +102,18 @@ export const validateBlockDirectory = (): void => {
         process.exit(1);
     }
 };
+
+export const validateIntegration = (
+    integrationName: string | undefined
+): number | undefined => {
+    if (!integrationName) {
+        return undefined;
+    }
+
+    const integration = INTEGRATIONS[integrationName!];
+    if (!integration) {
+        logError("Supported integrations are 'volt', 'standard', and 'v1'.");
```

Comment:
Unsupported category error puts each item on a separate line. Maybe map them from `INTEGRATIONS` keys sorted by ids but only listing the name.

Reply:
Good idea! 
![image](https://user-images.githubusercontent.com/346300/102803789-6e46b500-437e-11eb-9cef-1c3d42c87cc6.png)